Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```
@@ -1683,14 +1683,219 @@ script:
       required: true
     description: Retrieves attachments from a ticket.
     name: servicenow-get-ticket-attachments
-  dockerimage: demisto/python3:3.11.10.116949
+  - arguments:
+    - description: 'Short description of the ticket. Use issue fields as variables to enrich the text. Example: “${alert.name}”'
+      name: short_description
+      prettyname: Short Description
+      required: true
+    - auto: PREDEFINED
+      defaultValue: incident
+      description: Ticket type. Can be "incident", "problem", "change_request", "sc_request", "sc_task", "sc_req_item", "sn_si_incident" or "std_change_template". Default is "incident".
```

```suggestion
      description: Ticket type. 
```
no need to list the possible values or default value here.  It will appear in the README when you create it.